slotfills/sections/radio.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { PanelRow, RadioControl } from '@wordpress/components';
 import Field from '@/interfaces/field';
+import FMObject from '@/interfaces/fm-object';
 
 interface RadioProps {
   field: Field;
   index?: number;
   label?: string;
-  valueHook: (key: number | string) => [any | any[], Function];
+  valueHook: (key: number | string) => [string | FMObject | string[] | FMObject[], Function];
```

Comment:
I am seeing these possible values repeated in several places. Could you make an interface with these potential values and reduce the repetition needed to list these values out?

Reply:
maybe? As I learn more, I may refactor this.